### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ed1489**
extended description

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (koszek) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]: i dont like this

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (koszek) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: and this

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
```
Comment (koszek) [inline .github/ISSUE_TEMPLATE/feature_request.md:12]: and this also

**Review by koszek — CHANGES_REQUESTED** [review]
i left a couple of comments, please fix those

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ed1489) [reply .github/ISSUE_TEMPLATE/bug_report.md:11]: i think this is fine

**Review by ed1489 — COMMENTED** [review]
i made some changes, please review

**Review by koszek — APPROVED** [review]
ok, looks good

**Pull request merged**